test/rrule.test.ts

```diff
@@ -27,8 +27,8 @@ describe('RRule', function () {
     const s2 = rrulestr(s1).toString()
     expect(s1).equals(s2, s1 + ' => ' + s2)
   })
-  
-  it('rrulestr itteration not infinite when interval 0', function () {
```

Comment:
Haha nice! This should be submitted as its own PR to the original repository.

Reply:
Just a typo, the test was already here. I don't think i can be bothered submitting a 1-char PR.